Geometry/HcalCommonData/data/hcalSimNumbering/2023/hcalSimNumbering.xml

```diff
@@ -80,8 +80,9 @@
     22.0*cm, 165.0*cm, 30.0*cm, 0.0*cm, 1115.0*cm, 0.375*cm,
     1137.0*cm
   </Vector>
-  <Vector name="noff" type="numeric" nEntries="19">
-    15,103, 4, 10, 5, 2, 4, 18, 19, 11, 12, 13, 14,  3,  4, 53,  4,  0,  1
+  <Vector name="noff" type="numeric" nEntries="31">
```

Comment:
@kpedro88 - I'm not sure at what time you'd like to freeze 2023 scenarios and start file versioning.